libraries/botframework-schema/etc/botframework-schema.api.md

```diff
@@ -1716,6 +1716,54 @@ export interface TeamsMeetingInfo {
     id?: string;
 }
 
+// @public
+export interface TeamsMeetingNotification {
+    // (undocumented)
+    channelData: TeamsMeetingNotificationChannelData;
+    // (undocumented)
+    type: 'targetedMeetingNotification' | string;
+    // (undocumented)
+    value: TeamsMeetingNotificationInfo;
+}
+
+// @public
+export interface TeamsMeetingNotificationChannelData {
+    onBehalfOf?: TeamsMeetingOnBehalfOf;
+}
+
+// @public
+export interface TeamsMeetingNotificationInfo {
+    recipients: string[];
+    surfaces: TeamsMeetingNotificationSurface[];
+}
+
+// @public
+export interface TeamsMeetingNotificationRecipientFailureInfo {
+    errorCode: string;
+    failureReason: string;
+    recipientMri: string;
+}
+
+// @public
+export interface TeamsMeetingNotificationRecipientFailureInfos {
+    recipientsFailureInfo: TeamsMeetingNotificationRecipientFailureInfo[];
+}
+
+// @public
+export interface TeamsMeetingNotificationSurface {
+    content: TaskModuleContinueResponse;
+    contentType: 'task' | string;
+    surface: 'meetingStage' | string;
+}
```

Comment:
Currently `"contentType"` can only take on the value `task`. I decided to add `| string` so when a new possible content type in the future is released SDK changes don't necessarily have to be made for sdk users to use it. Same intent applies for other similar properties.